admin/backend/utils/base-decorator.js

```diff
@@ -23,7 +23,27 @@ class BaseDecorator {
   }
 
   getParent() {
-    return this.invokeOrGet('parentName') || this._resource.databaseName()
+    const parent = this.invokeOrGet('parent')
+    return {
+      name: this.getDatabaseName(parent),
+      icon: this.getIcon(parent)
```

Comment:
I would write this that way: 

```
getParent() {
  const parent = this.invokeOrGet('parent') || this._resource.databaseName()
  const name = parent.name || parent
  const icon = parent.icon ? `icon-${this._resource.databaseType() || 'database'}`

  return { name, icon }
}
```

and you don't need lines 33 - 46 anymore 

and put `mongodb` here https://github.com/SoftwareBrothers/admin-bro/pull/23/files#diff-0ad8f26ebf133da01b9d6c69805cd4fcR115